src/main/java/mate/academy/service/impl/MovieSessionServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.service.impl;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.MovieSessionDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.MovieSession;
+import mate.academy.service.MovieSessionService;
+
+@Service
+public class MovieSessionServiceImpl implements MovieSessionService {
+    @Inject
+    private MovieSessionDao movieSessionDao;
+
+    @Override
+    public MovieSession add(MovieSession movieSession) {
+        return movieSessionDao.add(movieSession);
+    }
+
+    @Override
+    public Optional<MovieSession> get(Long id) {
+        return Optional.ofNullable(movieSessionDao.get(id).orElseThrow(()
+                -> new DataProcessingException("Can`t get MovieSession: "
+                + id
+                + "by id", null)));
```

Comment:
The `get` method uses `orElseThrow` to throw a `DataProcessingException` with a `null` cause. Consider providing a more meaningful exception message or handling the absence of a movie session more gracefully.